packages/lit-dev-content/samples/tutorials/intro-to-lit/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+  "header": "Intro to Lit",
+  "difficulty": "Beginner",
+  "size": "small",
+  "duration": 20,
+  "category": "Learn",
+  "steps": [
+    {
+      "title": "Lit tutorial"
+    },
+    {
+      "title": "Define a component"
+    },
+    {
+      "title": "Properties and expressions"
+    },
+    {
+      "title": "Declarative event listeners"
+    },
+    {
+      "title": "More expressions"
+    },
+    {
+      "title": "Template logic"
+    },
+    {
+      "title": "Styles"
+    },
+    {
+      "title": "Finishing touches"
+    }
+  ]
+}
```

Comment:
Nit: trailing newline.

Files with trailing newlines work better with unix tools, I think it's a good default.

(Note there's an "Insert trailing newline" vscode setting to always do this.)